File: JAVA/Driving Institute exercise/com/Institute/Student.java
package com.Institute;

public class Student extends Person {
  
    private Plan plan;
    private String description;
    

    public Student(String names, String lastNames, int id, String residenseAddress, int phoneNumber, String rh, Plan plan) {
        super(names, lastNames, id, residenseAddress, phoneNumber, rh);
        this.plan=plan;
        this.description=description;
        
    }




    public Plan getPlan() {
        return this.plan;
    }

    @Override
    public String getDescription() {
         this.description = "Name: " + getNames() + " " + getLastNames() + "\n" +
                         "ID: " + getId() + "\n" +
                         "Address: " + getResidenceAddress() + "\n" +
                         "Phone Number: " + getPhoneNumber() + "\n" +
                         "RH: " + getRh();
    
       
        return description;
}  
    
}

File: PHP/Driving Institute exercise/Institute/Student.php
<?php

include_once 'Person.php';
include_once 'Plan.php';    

class Student extends Person {
    private string $description;
    private Plan $plan;

    public function __construct(string $names, string $lastNames, int $id, string $residenceAddress, int $phoneNumber, string $rh, Plan $plan) {
        parent::__construct($names, $lastNames, $id, $residenceAddress, $phoneNumber, $rh);
       
        $this->plan = $plan;
    }
    public function getPlan(): Plan {
        return $this->plan;
    }

    public function getDescription():string{
        $this->description= "Name: ".$this->getNames()." ".$this->getLastNames()."<br>".
                            "ID: ".$this->getId()."<br>".
                            "Address: ".$this->getResidenceAddress()."<br>".
                            "Phone: ".$this->getPhoneNumber()."<br>".
                            "RH: ".$this->getRh()."<br>";

        return $this->description;
    }

}